.github/issue-labeler.yml

```diff
@@ -1,24 +1,31 @@
+packages:create-discord-bot:
+  - "### Which package is this (bug report|feature request)
+    for\\?\\n\\ncreate-discord-bot"
 packages:brokers:
-  - '### Which package is this (bug report|feature request) for\?\n\nbrokers'
+  - "### Which package is this (bug report|feature request) for\\?\\n\\nbrokers"
```

Comment:
Have u tested this? iirc when I added it, I tried the same and it didn't worked

Reply:
I assume you're referring to this entire file. Yeah, it seems to work:

<img width="1275" alt="image" src="https://user-images.githubusercontent.com/33201955/233134423-9ea1a191-130c-4305-870c-580d3ec1970b.png">

<img width="659" alt="image" src="https://user-images.githubusercontent.com/33201955/233134393-87a1b7ab-504d-4213-a65f-c64cb5cf13ad.png">

Double quotes parse escapes hence the necessary extra backslashes.